core/services/health.go

```diff
@@ -176,7 +176,8 @@ func (c *checker) update() {
 	uptimeSeconds.Add(interval.Seconds())
 }
 
-func (c *checker) Register(name string, service Checkable) error {
+func (c *checker) Register(service Checkable) error {
```

Comment:
Out of tests, we only check `ServiceCtx`s, so every `Checkable` implementation already had a `Name()` method. Now that method is defined on `Checkable` instead of `ServiceCtx`.